lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
@marekj please remove the `this.elements` call as there is no need to query an element for this